packages/amplify-category-function/src/provider-utils/awscloudformation/utils/layerConfiguration.ts

```diff
@@ -24,7 +25,16 @@ export function getLayerConfiguration(layerName: string) {
 }
 
 export function getLayerRuntimes(layerName: string) {
-  return getLayerConfiguration(layerName).runtimes;
+  try {
+    return getLayerConfiguration(layerName).runtimes;
```

Comment:
Can we do the proper checks here instead of `try..catch`? like check file existence, read it, look for runtimes property, etc...

Reply:
I learned from @cjihrig not to do a file existence check, and instead use try/catch because it prevents a race condition.

I will add a check for the `runtimes` key though